src/style.css

```
@@ -1,3 +1,66 @@
-body {
+.body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  --blue-color: #00acdc;
+}
+
+.header {
+  width: auto;
+  display: flex;
+  justify-content: space-between;
+  align-content: center;
+  padding: 0 50px;
+}
+
+.logo__img {
+  margin: 10px 0;
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  display: flex;
+  margin-right: 20px;
+}
+
+.nav__item:last-child {
+  margin-right: 0;
+}
+
+.nav__link {
+  font-size: 12px;
+  font-weight: 500;
+  display: inline;
+  padding: 22.5px 0;
+  line-height: 14px;
+  text-decoration: none;
+  color: #000;
+  text-transform: uppercase;
+}
+
+.nav__link:hover {
+  color: var(--blue-color);
+}
+
+.is-active {
+  color: var(--blue-color);
+}
+
+.is-active::after {
+  position: relative;
+  content: "";
+  display: block;
+  top: 19px;
+  bottom: 0;
+  /* width: 100%; */
```

```suggestion
  position: absolute;
  content: "";
  left: 0;
  bottom: 0;
  width: 100%;
```
pseudo element should be with positon absolute and be positioned to closest non static element.
we can avoid to hardcode top property.
set for is-active position: relative.